### PR TITLE
Add Kitchen and Travis

### PR DESCRIPTION
**Pull request by Martin819**

### CODE REVIEW

File: tests/pillar/cluster.sls [hunk 1]
```diff
@@ -25,6 +25,7 @@ horizon:
       host: 127.0.0.1
       port: 5000
       api_version: 2
+      encryption: encryption
```
Comment (fpytloun) [inline tests/pillar/cluster.sls]: This should be set to "ssl", not "encryption"

**Comment by Martin819** [comment]
"encryption: ssl" corrected on both pillars

**Comment by epcim** [comment]
I enabled travis, wait to finish build.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,9 @@ tests/build/
 *.swp
 *.pyc
 .ropeproject
+.kitchen
+.bundle
+.vendor
+Gemfile*
+vendor/
+kitchen-init.sh
```
Comment (epcim) [inline .gitignore]: this file should never be there..

File: .kitchen.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+driver:
+  name: docker
+  hostname: horizon.ci.local
+  use_sudo: false
+
```
Comment (epcim) [inline .kitchen.yml:6]: whitespaces, fix (latest kitchen_init.sh on upstream is already fixed)

File: .kitchen.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  pillars:
+    top.sls:
+      base:
+        "*":
+          - horizon
+
+verifier:
+  name: inspec
+  sudo: true
+platforms:
```
Comment (epcim) [inline .kitchen.yml]: add space

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    gem 'kitchen-salt', :git => 'https://github.com/epcim/kitchen-salt.git', :branch => 'dependencis-pkg-repo2'
+    #Waiting for PR#78
+    #gem 'kitchen-salt', '>=0.2.25'
+  - bundle install
+
+before_script:
+  - make test
+
+script:
+  - test ! -e .kitchen.yml || bundle exec kitchen test
```
Comment (epcim) [inline .travis.yml]: I suggest to add gitter/slack notification settings to default travis. Especially disable failed builds to be notified globally/owners (ie: only to commit author)

**Review by epcim — CHANGES_REQUESTED** [review]
I would keep only single or cluster test. They don't differ much. Anyway we test only on single instance.

Cluster is worth to check iteration over dict entries. That will be anyway tested in upper stage.

**Comment by epcim** [comment]
@fpytloun pls consider globally to update run_tests with: https://github.com/salt-formulas/salt-formula-opencontrail/pull/2/files#diff-6734f3f06fbfda089cd1d6eef82301caR47

**Pull request merged**